runner/src/main/java/com/mulesoft/tools/migration/MigrationRunner.java

```diff
@@ -158,6 +166,16 @@ private void initializeOptions(String[] args) {
 
       }
 
+      if (line.hasOption(PROJECT_PARENT_GAV)) {
+        final String value = line.getOptionValue(PROJECT_PARENT_GAV);
+        if (StringUtils.isNotEmpty(value) && value.split(":").length == 3) {
+          this.projectParentGAV = new ParentBuilder().withGroupId(value.split(":")[0]).withArtifactId(value.split(":")[1])
+              .withVersion(value.split(":")[2]).build();
```

Comment:
There's no need to split value variable several times. Please change by 
```
  String[] gav = value.split(":");
  this.projectParentGAV = new ParentBuilder()
      .withGroupId(gav[0])
      .withArtifactId(gav[1])
      .withVersion(gav[2])
      .build();
```

Reply:
you are right. removed